src/global-firestore-data/user-semesters.ts

```diff
@@ -12,6 +12,18 @@ import {
   deleteCoursesFromRequirementChoices,
 } from './user-overridden-fulfillment-choices';
 
+export const editCollections = async (
+  updater: (oldCollections: readonly Collection[]) => readonly Collection[]
+): Promise<void> => {
+  console.log('edit Collections');
+  const collections = updater(store.state.collections);
+  store.commit('setCollections', collections);
+  await updateDoc(doc(semestersCollection, store.state.currentFirebaseUser.email), {
+    collections,
+  });
+  store.commit('setOrderByNewest', store.state.orderByNewest);
```

Comment:
This shouldn't be necessary here since editing collections should not change the ordering of the semesters